apps/fihi-app/src/main.tsx

```diff
@@ -1,6 +1,7 @@
 import React from "react";
 import ReactDOM from "react-dom/client";
 import App from "./app/layout";
+import "@incmix/ui/styles/global.css"
```

Comment:
Moved the css file to the 'up most' file as well. I don't thing this changes anything in behaviour, but it's just a pattern I like to use.